django/db/models/sql/query.py

```diff
@@ -1121,6 +1121,8 @@ def build_filter(self, filter_expr, branch_negated=False, current_negated=False,
         query. However, if the filter isn't added to the query then the caller
         is responsible for unreffing the joins used.
         """
+        if isinstance(filter_expr, dict):
```

Comment:
Ah, I think I forgot my parens when checking. Thanks for double-checking for me :)